google-cloud-tasks/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Release History
 
+### 0.2.5 / 2018-10-18
```

Comment:
I know this patch version change is established practice for "lower-level" additions in this project, but it just seems strange to me not to increment the minor version when upgrading the service API version.